Sources/Moya/MoyaProvider.swift

```diff
@@ -77,30 +82,37 @@ open class MoyaProvider<Target: TargetType> {
     /// Designated request-making method. Returns a `Cancellable` token to cancel the request later.
     @discardableResult
     open func request(_ target: Target, completion: @escaping Moya.Completion) -> Cancellable {
-        return self.request(target, queue: nil, completion: completion)
+        return self.request(target, callbackQueue: callbackQueue, completion: completion)
     }
 
-    /// Designated request-making method with queue option. Returns a `Cancellable` token to cancel the request later.
+    /// Designated request-making method with an optional callback queue. Returns a `Cancellable` token to cancel the request later.
     @discardableResult
-    open func request(_ target: Target, queue: DispatchQueue?, progress: Moya.ProgressBlock? = nil, completion: @escaping Moya.Completion) -> Cancellable {
-        return requestNormal(target, queue: queue, progress: progress, completion: completion)
+    open func request(_ target: Target, callbackQueue: DispatchQueue?, progress: Moya.ProgressBlock? = nil, completion: @escaping Moya.Completion) -> Cancellable {
```

Comment:
You are right, default argument seems like a good idea. I'm thinking of `nil`, since in case of a nil `self.callbackQueue` is taken anyways. What do you think?